KsApi/models/graphql/adapters/User+GraphUser.swift

```diff
@@ -22,6 +22,7 @@ extension User {
       location: nil,
       name: graphUser.name,
       needsFreshFacebookToken: nil,
+      needsPassword: false,
```

Comment:
Yea so I think you can remove this entire file and its' test file, no longer being used, all user creation is being done with the `User+UserFragment` it seems like. I double checked it by removing it from the target membership and the project still built.

Reply:
ok. will remove